### ISSUE TITLE
pdfviewer: custom stamps with png image breaks the service api

### ISSUE THREAD

**Issue by bort2012** [syncfusion/ej2-javascript-ui-controls#96]
I have created a custom stamp like this:
`this.pdfviewerControl.customStampSettings.customStamps = [{ customStampName: "Signature", customStampImageSource: this.settingsService.signatureBase64 }];`

The base64 string is from a png with transparent background.
When I add the custom stamp to the pdf it works and places the image with transparent background. 
When I download the pdf the custom stamp is shown as expected.
But when I use the saveAsBlob method to save the pdf into my database something goes wrong:
When I load the saved pdf again the custom stamp is not shown but the annotation is visible in the annotation comment box. Then when I want to run saveAsBlob again it says: The service api is not responding.
If I do all this with a jpg custom stamp everything works.

I use your docker-image for the pdfviewer-service-api.

I call the saveAsBlob mehtod like this:

```
 const reader = new FileReader();
        this.pdfviewerControl.saveAsBlob().then((blob) => {
          reader.readAsDataURL(blob);
          reader.onloadend = () => {
            const base64data = reader.result;
            // base64data goes to my db
          });
```
I have had a look into the annotations array and in the case of my custom png stamp there is no imageSrc and no author etc. after reloading from the db. 
In the case of jpg the imageSrc of the annotation is there after reloading from the db. But the strange thing is that it is now shown as base64 type png although the origin was jpg.

**Comment by bort2012** [comment]
are there any news?

**Comment by DhivyaBharathi1307** [comment]
Hi Team,

We suspect that the reported issue occurs due to that Syncfusion PDF Viewer does not have support to render the PNG images in our older versions. However, currently, we have provided support to render the PNG images in our PDF Viewer. We suggest you use our latest version v19.4.0.48 for both the Client and server side to get the issue resolved. We have shared the sample in the latest version.


Sample: https://stackblitz.com/edit/pt2jru?file=index.ts 


Kindly try this and let us know whether the reported issue is resolved or not. If the issue is still not resolved, kindly share the base64 string of the custom stamp image in which you are trying to add the custom stamp in the PDF Viewer. It will be helpful for us to investigate further and assist you better.

Regards,
Dhivya.
